src/workloads/execution/PipelineUpdate.yml

```diff
@@ -0,0 +1,102 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: 2000
+    Document: &Document
+      t: &partitionID {^RandomInt: {min: 0, max: 100}}
+      randomID: &randomID {^RandomInt: {min: 0, max: 10000}}
+      int: &integer {^RandomInt: {min: 0, max: 100}}
+      str: &string {^RandomString: {length: {^RandomInt: {min: 20, max: 50}}}}
+      largeStr: &largeString {^RandomString: {length: 1024}}
+      obj: &object {a: *integer, b: *integer, c: *string}
+      arr: &array [*integer, *string, *object, *largeString]
+      nestedObj: &nestedObject {
+        int: *integer,
+        str: *string,
+        obj: *object,
+        arr: [1, 2, 3],
+        nestedArr: [*integer, *string, *object, *array],
+        largeStr: *largeString}
+    Indexes:
+    - keys: {t: 1}
```

Comment:
Good call to have this index. We don't want a lot of time in the workload being spent doing collection scans.